contracts/ssnlist.scilla

```diff
@@ -606,7 +636,6 @@ transition UpdateDeleg(ssnaddr : ByStr20, deleg : ByStr20, stake_amt : Uint128,
       | None =>
       end
     | False =>
```

Comment:
should deposit into buffered deposit first

Reply:
> should deposit into buffered deposit first

I think `UpdateDeleg` is only used for contract upgrading right? So we should use other help functions e.g `PopulateBuffDeposit` or `PopulateDirectDeposit` to recovery the state data just like before upgrading.

Reply:
ok